confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -139,17 +153,22 @@ def register(self, subject, avro_schema):
         schema_id = result['id']
         # cache it
         self._cache_schema(avro_schema, schema_id, subject)
+
         return schema_id
 
-    def get_by_id(self, schema_id):
+    def get_by_id(self, schema_id, json_format=False):
```

Comment:
is json_format really necessary? It seems that HAS_FAST would be sufficient and already covers all the code paths where they are both true.